posthog/tasks.py

```diff
@@ -10,12 +10,14 @@ def add(x, y):
     return x + y
 
 @shared_task
-def post_event_to_slack(event_id):
+def post_event_to_slack(event_id, site_url):
```

Comment:
Would be good to add types here

Reply:
Also just thinking about it, does it make sense to just pass through the entire Event object rather than ID? I don't know if that's going to be inefficient.

We definitely want to move the entire capture.py stack to workers so we could leave for now

Reply:
Types added. 

The event would get serialised as json and then unserialised on the other end and pass through redis like that. I don't know if it would slow down things or if something would get lost in transmission, however what can happen is that we end up with stale data on the other end.

For example, when eventually moving the event<->action matching to workers, I'd suggest adding a `processed` or similar bool to Event in order to 1) not process the same thing twice and 2) be more resilient in case some queued tasks get lost (with a "fix" task that on startup processes all events that have processed=False).